### PR TITLE
feat: add default group_id

### PR DESCRIPTION
**Pull request by florianriche**
When we don't specify a group_id in inpout when creating a site, it yields an exception.
This PR adds a check that:
1/ The requester is allowed to specify a group (is admin or belongs to the group_id set in the payload)
2/ Add the group_id of the requester in the payload if the group_id is not specified.

This is done for both the standard site creation and the noAlert sites.

### CODE REVIEW

File: src/tests/routes/test_sites.py [hunk 1]
```diff
@@ -97,9 +97,9 @@ async def test_fetch_sites(test_app_asyncio, init_test_db):
          201, None],
         [1, {"name": "my_site", "group_id": 1, "lat": 0., "lon": 0.,
              "country": "FR", "geocode": "01"}, True, 201, None],
-        [0, {"name": "my_site", "group_id": 1, "lat": 0., "lon": 0.,
+        [0, {"name": "my_site", "group_id": 2, "lat": 0., "lon": 0.,
              "country": "FR", "geocode": "01"}, False,
-         401, "Permission denied"],
+         401, "You can't specify another group"],
```
Comment (frgfm) [inline src/tests/routes/test_sites.py:102]: Sorry, I'm not sure I understand the logic: did the test case change?

If possible, I'd rather keep the group_id 1 and change the expected outcome if it did, and then adding a test case for group_id 2

**Review by frgfm — CHANGES_REQUESTED** [review]
Thanks for the PR! I added a few comments

File: src/tests/routes/test_sites.py (hunk 1 above)
Comment (florianriche) [reply src/tests/routes/test_sites.py:105]: Done

File: src/app/api/routes/sites.py [hunk 2]
```diff
@@ -15,23 +16,37 @@
 
 
 @router.post("/", response_model=SiteOut, status_code=201, summary="Create a new site")
-async def create_site(payload: SiteIn, _=Security(get_current_access, scopes=[AccessType.admin])):
+async def create_site(payload: SiteIn,
+                      requester=Security(get_current_access, scopes=[AccessType.admin, AccessType.user])):
```
[the remark this answers is not in the document]
Comment (florianriche) [reply src/app/api/routes/sites.py]: Done

**Review by fe51 — APPROVED** [review]
Thanks for the PR !

**Pull request merged**